### PR TITLE
[7.5.x] JBPM-6794 - Invalid content was found starting with element caused by…

### PR DESCRIPTION
**Pull request by tsurdilo**
… 'documentation' ordering (#740)

### CODE REVIEW

**Comment by tsurdilo** [comment]
depends on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/712

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/util/BaseElementFeaturesTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
```
Comment (LuboTerifaj) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/util/BaseElementFeaturesTest.java]: Can you please change year to 2018?

**Comment by tsurdilo** [comment]
@LuboTerifaj I think we should do this across the board and not just in this one file. I would leave this as is as the license header update is typically afaik a across-module global change that we do.

**Comment by LuboTerifaj** [comment]
@tsurdilo I thought that when it is a new code, it could be right value. But I am ok with old value as well. Thanks!

**Pull request merged**